integration/spark/src/main/scala/org/apache/carbondata/geo/GeoUtilUDFs.scala

```diff
@@ -30,6 +32,7 @@ object GeoUtilUDFs {
     sparkSession.udf.register("LatLngToGeoId", new LatLngToGeoIdUDF)
     sparkSession.udf.register("ToUpperLayerGeoId", new ToUpperLayerGeoIdUDF)
     sparkSession.udf.register("ToRangeList", new ToRangeListUDF)
+    sparkSession.udf.register("ToRangeListAsString", new ToRangeListAsStringUDF)
```

Comment:
these UDF are exposed to user right ? can we update in the document ?

Reply:
This udf is not exposed to user. This is for internal purpose only.